narwhals/_pandas_like/series_dt.py

```diff
@@ -242,3 +245,55 @@ def truncate(self, every: str) -> PandasLikeSeries:
         return self.with_native(
             self.native.dt.floor(f"{multiple}{UNIT_DICT.get(unit, unit)}")
         )
+
+    def offset_by(self, by: str) -> PandasLikeSeries:
+        import pandas as pd
```

Comment:
it should be fine to import this at the top of the file, the pandas-like libraries all depend on pandas anyway